### PR TITLE
AZ-1269: New detector for VPN Tunnels' FAME Query

### PR DESCRIPTION
**Pull request by jmapro**
- [x] Depend on internal MR on RUN module.

### CODE REVIEW

File: modules/fame_azure-vpn/conf/02-tunnel-status.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module: Azure VPN
+name: IPSEC Tunnel Status
+id: tunnelstatus
+
+transformation: ".fill()"
```
Comment (BzSpi) [inline modules/fame_azure-vpn/conf/02-tunnel-status.yaml:5]: It seems to me that `fill` without value is useless.
Also, use extrapolation instead.

File: modules/fame_azure-vpn/conf/02-tunnel-status.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+aggregation: ".mean(by=['azure_resource_group', 'azure_resource_name', 'remote_ip'])"
+
+signals:
+  signal:
+    metric: fame.azure.virtual_network_gateway.tunnel_status
+
```
Comment (BzSpi) [inline modules/fame_azure-vpn/conf/02-tunnel-status.yaml:11]:
```suggestion
    metric: fame.azure.virtual_network_gateway.tunnel_status
    extrapolation: 'zero'
```

File: modules/fame_azure-vpn/conf/02-tunnel-status.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+rules:
+  critical:
```
Comment (BzSpi) [inline modules/fame_azure-vpn/conf/02-tunnel-status.yaml:13]: No major alert ? With a smaller lasting for example.

File: modules/fame_azure-vpn/conf/02-tunnel-status.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+module: Azure VPN
+name: IPSEC Tunnel Status
+id: tunnelstatus
```
Comment (BzSpi) [inline modules/fame_azure-vpn/conf/02-tunnel-status.yaml:3]:
```suggestion
id: tunnel_status
```

**Review by BzSpi — CHANGES_REQUESTED** [review]

File: modules/fame_azure-vpn/conf/02-tunnel-status.yaml (hunk 2 above)
Comment (BzSpi) [reply modules/fame_azure-vpn/conf/02-tunnel-status.yaml:11]: use fill instead to have better control of the duration.

**Review by BzSpi — APPROVED** [review]

File: modules/fame_azure-vpn/conf/02-tunnel-status.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+rules:
+  critical:
+    threshold: 0
+    comparator: "=="
+    lasting_duration: '20m'
+  major:
+    threshold: 0
+    comparator: "=="
+    lasting_duration: '10m'
```
Comment (haedri) [inline modules/fame_azure-vpn/conf/02-tunnel-status.yaml:20]: It's missing a dependency here I think

**Pull request merged**